src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -854,7 +858,19 @@ private JCheckBox getCheckBoxSingleCookeRequestHeader() {
         return checkBoxSingleCookieRequestHeader;
     }
 
-    private ProxyExcludedDomainsMultipleOptionsPanel getProxyExcludedDomainsPanel() {

```

Comment:
This indentation change should be reverted (same for above unnecessary changes).